iwyu_test_util.py

```diff
@@ -92,6 +134,32 @@ def HasTarget(target):
         supported.append(m.group(1))
     return target in supported
 
+  @staticmethod
+  def UsingCXXstdlib(lib):
+    id_macro = {
+      'libcxx' : '_LIBCPP_VERSION',
+      'libstdcxx' : '__GLIBCXX__',
+      'msvc' : '_MSVC_STL_UPDATE', # https://github.com/microsoft/STL/wiki/Macro-_MSVC_STL_UPDATE
+    }
+    if lib not in id_macro:
+      assert False, ('Unknown STL: %s' % lib)
+
+    return Features._CheckForCXXPreprocessorMacro(id_macro[lib])
+
+  @staticmethod
+  def UsingCstdlib(lib):
+    id_macro = {
+      'glibc' : '__GLIBC__',
+      'macos' : '__APPLE_CC__', # This identifies the Apple compiler, is this good enough?
+      'msvc' : '_MSVC_VER', # This identifies the MS compiler, is this good enough?
```

Comment:
The compiler-identifying symbol for MSVC actually `_MSC_VER`. And yeah, I think it's good enough!